### PR TITLE
Lb pomodoro user dto refactoring

### PR DESCRIPTION
**Pull request by b-laci**

### CODE REVIEW

File: service/src/main/java/eu/execom/labs/pomodorospringreact/utils/LazyCollectionInitStrategy.java [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package eu.execom.labs.pomodorospringreact.utils;
+
+public enum LazyCollectionInitStrategy {
```
Comment (dejanjesic) [inline service/src/main/java/eu/execom/labs/pomodorospringreact/utils/LazyCollectionInitStrategy.java]: Maybe this enum should be inside of UserCollectionInitializer class?

File: service/src/main/java/eu/execom/labs/pomodorospringreact/utils/UserCollectionInitializer.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
+package eu.execom.labs.pomodorospringreact.utils;
+
+import eu.execom.labs.pomodorospringreact.entity.User;
+
+public abstract class UserCollectionInitializer {
```
Comment (dejanjesic) [inline service/src/main/java/eu/execom/labs/pomodorospringreact/utils/UserCollectionInitializer.java]: Maybe the better solution here is to provide private constructor, if it's abstract someone may think that it's made for extension.

Comment (idostanic) [reply service/src/main/java/eu/execom/labs/pomodorospringreact/utils/UserCollectionInitializer.java]: indeed why is this class abstract? @b-laci

Comment (b-laci) [reply service/src/main/java/eu/execom/labs/pomodorospringreact/utils/UserCollectionInitializer.java]: you are right i will remove abstract, and will go with the private constructor

**Pull request merged**